cms/static/cms/js/modules/cms.plugins.js

```diff
@@ -509,7 +514,82 @@
                     e.stopPropagation();
                 });
 
-                dropdown.find('a').on('click.cms', function (e) {
+                that._setupActions(nav);
+                // prevent propagnation
+                nav.on(this.click + ' pointerup.cms pointerdown.cms click.cms dblclick.cms', function (e) {
```

Comment:
please keep in mind on cleanup:

- Move all elements to ``this.ui`` (search for .find() and replace)
- add YUIdocs to general outline
- set the name for anonymous functions: ``_events: function () {`` becomes ``_events: function _events() {``
- set event handler variables like ``this.pointerup = 'pointerup.cms.sideframe pointercancel.cms';``
- replace ``bind`` with ``on`` and ``unbind`` with ``off``
- ensure for correct variable name positioning
- remove as many timeouts as possible